test/parallel/test-eventtarget.js

```diff
@@ -432,10 +432,21 @@ ok(EventTarget);
   target.removeEventListener('foo', a, { capture: false });
   target.dispatchEvent(new Event('foo'));
 }
-
 {
   const target = new EventTarget();
   strictEqual(target.toString(), '[object EventTarget]');
   const event = new Event('');
   strictEqual(event.toString(), '[object Event]');
 }
+{
+  strictEqual(Event.NONE, 0);
+  strictEqual(Event.CAPTURING_PHASE, 1);
+  strictEqual(Event.AT_TARGET, 2);
+  strictEqual(Event.BUBBLING_PHASE, 3);
+  strictEqual(new Event().eventPhase, Event.NONE);
```

Comment:
Missing required event type ...
```suggestion
  strictEqual(new Event('foo').eventPhase, Event.NONE);
```